precompile/testutils/test_predicate.go

```diff
@@ -0,0 +1,87 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package testutils
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ava-labs/subnet-evm/precompile/precompileconfig"
+	"github.com/stretchr/testify/require"
+)
+
+type PredicateTest struct {
+	Config precompileconfig.Config
+
+	ProposerPredicateContext *precompileconfig.ProposerPredicateContext
+
+	StorageSlots         []byte
+	Gas                  uint64
+	GasErr, PredicateErr error
+}
+
+func (test PredicateTest) Run(t testing.TB) {
+	t.Helper()
+	var (
```

Comment:
can we add `require := require.New(t)` and reduce the number of arguments for all require

Reply:
yup